frontend/src/components/uam/uam.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useAppDispatch, useAppSelector } from 'hooks/hooks';
+import { FC } from 'common/types/types';
+import { UsersTable } from './users-table/users-table';
+import { usersActions } from 'store/actions';
+import { usersColumns } from './common/users-columns';
+import styles from './styles.module.scss';
+
+const Uam: FC = () => {
+  const dispatch = useAppDispatch();
+  const { users } = useAppSelector((state) => state.users);
+
+  useEffect(() => {
+    dispatch(usersActions.getUsers());
+  }, []);
+
+  return (
+    <div className={styles.uam}>
+      <UsersTable data={users ?? []} columns={usersColumns} />
```

Comment:
```suggestion
   const columns = getColumns();
   const rows = getRows(users);

      <UsersTable data={rows} columns={columns} />
```

Reply:
What should these getColumns() and getRows() functions do if table works with usersColumns and users well?